lib/paths.cf

```diff
@@ -315,7 +315,7 @@ bundle common paths
       "path[tr]"            string => "/usr/bin/tr";
       "path[wc]"            string => "/usr/bin/wc";
       "path[wget]"          string => "/usr/bin/wget";
-      "path[realpath]"      string => "/usr/bin/realpath";
+      "path[realpath]"      string => "/usr/bin/readlink -e";
```

Comment:
Actually I just thought of something this won't work well with.

We define classes based on the presence of the path. `/usr/bin/readline[space]-e` doesn't exist.

I think you might be interested in https://tracker.mender.io/browse/CFE-1804?jql=text%20~%20%22paths%22

I think instead of swapping realpath like that and masking what it really is, it would be better to add the path for readlink.

Thoughts?